template.tpl

```diff
@@ -1403,30 +1404,42 @@ ___TEMPLATE_PARAMETERS___
                   },
                   {
                     "value": "selfHosted",
-                    "displayValue": "Self-hosted",
-                    "subParams": [
-                      {
-                        "help": "Add the URL where your self-hosted Snowplow JavaScript library can be downloaded from. If this location is not in AWS S3 (via Cloudfront) or GCP Storage, remember to update \u003ca href\u003d\"https://www.simoahava.com/analytics/custom-templates-guide-for-google-tag-manager/#injects-scripts\"\u003e\u003cstrong\u003etemplate permissions\u003c/strong\u003e\u003c/a\u003e to allow script injection requests to this URL.",
-                        "valueValidators": [
-                          {
-                            "args": [
-                              "^https://.*"
-                            ],
-                            "type": "REGEX"
-                          }
-                        ],
-                        "displayName": "Self-hosted Library URL",
-                        "simpleValueType": true,
-                        "name": "selfHostedUrl",
-                        "type": "TEXT",
-                        "valueHint": "https://123.cloudfront.net/sp.js"
-                      }
-                    ]
+                    "displayValue": "Self-hosted"
+                  },
+                  {
+                    "value": "doNotLoad",
+                    "displayValue": "Do not load library"
                   }
                 ],
                 "simpleValueType": true,
-                "help": "Load the Snowplow JavaScript library (latest version) from a CDN or choose the location where the self-hosted library can be loaded from.",
-                "defaultValue": "selfHosted"
+                "defaultValue": "selfHosted",
+                "help": "Load the Snowplow JavaScript library from a third-party CDN or choose the location where the self-hosted library can be loaded from."
```

Comment:
Wonder if this can be extended to mention what `doNotLoad` means and _very briefly_ explain why you might not load it here e.g. "Do not load can be used when the Tracker Snippet is loaded with another technique such as directly on the page"

Reply:
Added in help text!